File: Algoritmos Traduzidos/Perl/Algoritmos_Hugo/46.pl
#Fazer um algoritmo que possa entrar com o saldo de uma aplicação e imprima o novo saldo, considerando o reajuste de 1%.

print "introduza o saldo:\n";
$saldo = <STDIN>;
$nsaldo = $saldo * 1.01;
print "o novo saldo e: $nsaldo\n";

File: Algoritmos Traduzidos/Ruby/Algoritmos_Hugo/46.rb
#Fazer um algoritmo que possa entrar com o saldo de uma aplicação e imprima o novo saldo, considerando o reajuste de 1%.
puts "introduza o saldo:"
saldo = gets
nsaldo = saldo.to_f * 1.01
puts "o novo saldo e: #{nsaldo}"